components/trello/trello.app.js

```diff
@@ -0,0 +1,240 @@
+const axios = require("axios");
+const events = `addAttachmentToCard
+addChecklistToCard
```

Comment:
Since we're displaying this to users, it might be nice to present human-readable options to the user:

```
[
  { label: "Add Checklist To Card", value: `addChecklistToCard` }
  ...
]
```